stable/artifactory-ha/README.md

```diff
@@ -729,6 +729,20 @@ This can be done with the following parameters
 --set database.password=${DB_PASSWORD} \
 ...
 ```
+```bash
```

Comment:
Can you convert the long command line to a `values-oracle.yaml` snippet? I would rather users adopt the method of creating a custom `values-xyz.yaml` and passing it with a simple
```
helm upgrade --install artifactory-ha -f values-xyz.yaml jfrog/artifactory-ha
```
Cleaner and easier to digest...
 
Same for the `artifactory` README.

Reply:
Now its changed to yaml snippet.